test/ci_util.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+##############################################################################
+#
+# This script gathers the output from the $SCM_ROOT/ccpp-scm/scm/run directory
+# for the SCM CI Regression Tests. 
+#
+##############################################################################
+import os
+import sys
+from rt_test_cases import run_list
+from os.path import exists
+import argparse
+
+#
+parser = argparse.ArgumentParser()
+parser.add_argument('-b', '--build_type',  help='SCM build type')
+
+def parse_args():
+    args       = parser.parse_args()
+    build_type = args.build_type
+    return (build_type)
+
+def main():
+    
+    (build_type) = parse_args()
+
+    #
+    for run in run_list:
+        file_out = "../scm/run/" + "output_"+run["case"]+"_"+run["suite"]+"/output.nc"
```

Comment:
Looks like there is room for defining some common strings to use to clean this up a bit (`run["case"]+"_"+run["suite]`...)